airflow/providers/cncf/kubernetes/utils/pod_manager.py

```diff
@@ -721,14 +721,27 @@ def read_pod(self, pod: V1Pod) -> V1Pod:
         except HTTPError as e:
             raise AirflowException(f"There was an error reading the kubernetes API: {e}")
 
-    def await_xcom_sidecar_container_start(self, pod: V1Pod) -> None:
+    def await_xcom_sidecar_container_start(self, pod: V1Pod, startup_timeout: int = 120) -> None:
```

Comment:
Can we add test cases for this change? Like basically call this, sleep for certain seconds and then check if the exception came up